aopy/preproc/base.py

```diff
@@ -535,6 +535,7 @@ def get_trial_segments_and_times(events, times, start_events, end_events):
         times (nt): times vector
         start_events (list): set of start events to match
         end_events (list): set of end events to match
+        repeating_start_events (bool): whether the events used as start events may occur multiple times within one segment
```

Comment:
```suggestion
        repeating_start_events (bool): whether the start events might occur multiple times within one segment. Otherwise always use the last start event within a segment. May lead to segments spanning multiple trials if used improperly. Default False.
```